pkg/process/config/config_test.go

```diff
@@ -341,7 +340,6 @@ func TestAgentConfigYamlAndSystemProbeConfig(t *testing.T) {
 	if runtime.GOOS != "windows" {
 		assert.Equal("/var/my-location/system-probe.log", agentConfig.SystemProbeAddress)
 	}
-	assert.Equal(append(processChecks, ConnectionsCheckName, NetworkCheckName), agentConfig.EnabledChecks)
```

Comment:
was this test invalid since it was asserting the presence of `NetworkCheckName` ?

Reply:
Yes, that and also the fact that the config package has no real way of calling `getChecks`, so I figured it made the most sense to remove it. If you think it's worthwhile to keep, I could move `getChecks` to `pkg/process/config`, or `pkg/config` (since importing `cmd/process-agent` would cause an import loop, which go explicitly disallows)

Reply:
There's no problem tbh. I was just double checking because I saw some tests that we're removing that use  `NetworkCheckName` and `OOMKillCheckName` although processa-agent doesn't run them.

Reply:
Ah, okay cool! Just making sure